handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -89,6 +94,24 @@
             destroyedUpdater = AtomicIntegerFieldUpdater.newUpdater(OpenSslEngine.class, "destroyed");
         }
         DESTROYED_UPDATER = destroyedUpdater;
+
+        Method getUseCipherSuitesOrderMethod = null;
+        Method setUseCipherSuitesOrderMethod = null;
+        if (PlatformDependent.javaVersion() >= 8) {
+            try {
+                getUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("getUseCipherSuitesOrder");
+                SSLParameters parameters = new SSLParameters();
+                Boolean order = (Boolean) getUseCipherSuitesOrderMethod.invoke(parameters);
```

Comment:
@Scottmitch just tested and we can not cast to boolean. Needs to be an Object so Boolean is the way to go.